src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -27,14 +27,24 @@
 
     public static final String VALID_NAME_AMY = "Amy Bee";
     public static final String VALID_NAME_BOB = "Bob Choo";
+    public static final String VALID_NAME_CEDAR = "Cedar Mush";
+    public static final String VALID_NAME_DELON = "Delon Fattie";
     public static final String VALID_PHONE_AMY = "11111111";
     public static final String VALID_PHONE_BOB = "22222222";
+    public static final String VALID_PHONE_CEDAR = "33333333";
+    public static final String VALID_PHONE_DELON = "444444444";
     public static final String VALID_EMAIL_AMY = "amy@example.com";
     public static final String VALID_EMAIL_BOB = "bob@example.com";
+    public static final String VALID_EMAIL_CEDAR = "cedar@example.com";
+    public static final String VALID_EMAIL_DELON = "delon@example.com";
     public static final String VALID_ADDRESS_AMY = "Block 312, Amy Street 1";
     public static final String VALID_ADDRESS_BOB = "Block 123, Bobby Street 3";
+    public static final String VALID_ADDRESS_CEDAR = "Block 123, Cedary Street 3";
+    public static final String VALID_ADDRESS_DELON = "Block 123, Dragon Street 3";
     public static final String VALID_TAG_HUSBAND = "husband";
     public static final String VALID_TAG_FRIEND = "friend";
```

Comment:
Update: I have taken a closer look at the tests, and have concluded that *`Cedar` and `Delon` are still unfortunately redundant* -- please remove them since they add no value to the tests :(

Explanation:
Observe that you can simply add `Amy` and `Bob` into the list returned by `getTypicalPersons()` method in `TypicalPersons`and all your tests will still pass. The reason why they did not pass at first without `Amy`, `Bob`, `Cedar` and `Delon` is just because `VALID_TAG_FRIEND` is stated to be `friend`, not `friends`, so it throws an exception claiming that the tag does not exist in the original list given by the original `getTypicalPersons()` method.

Reply:
Update 2: Upon more meticulous testing (this time by running `./gradlew allTests` and waiting some 22 minutes), I propose a superior alternative to resolving this issue: replace `VALID_TAG_FRIEND` with `friends` in `RemoveTagCommandTest.java`. This will prevent the need to edit `TypicalPersons` list, while still ensuring that the tests are all carried out properly. Alternatively, you can create a `VALID_TAG_FRIENDS` in `CommandTestUtil` instead